service/history/replication/task_processor.go

```diff
@@ -430,10 +430,12 @@ func (p *taskProcessorImpl) processSingleTask(replicationTask *types.Replication
 			tag.Error(err),
 		)
 		//TODO: uncomment this when the execution fixer workflow is ready
-		//if err = p.triggerDataInconsistencyScan(replicationTask); err != nil {
-		//	p.logger.Warn("Failed to trigger data scan", tag.Error(err))
-		//	p.metricsClient.IncCounter(metrics.ReplicationDLQStatsScope, metrics.ReplicationDLQValidationFailed)
-		//}
+		// Uncommenting this to verify that the scanner indeed gets triggered.
+		if err = p.triggerDataInconsistencyScan(replicationTask); err != nil {
```

Comment:
I think we should add a flipr control over it, because it might add lots of extra load.

Reply:
Good idea! Will add a flipr for this. Thanks for the suggestion.